src/routes_student.py

```diff
@@ -112,7 +112,7 @@ def student_get_run_status(netid, cid, aid, run_id):
 			if not verify_student_or_staff(netid, cid):
```

Comment:
Can we also rename this function to `student_get_job_status` to be consistent with the one on the staff side? (That was changed to `staff_get_job_status`)

Reply:
Sounds good